bcgov/HOUS-permit-portal
# Fix issue inviting reviewers with no name

## Description

User first/last name is required by the UI once the user has access to the system. This was previously handled on the accept invitation screen but that only covers the case where the user creates an account manually. If the user uses BCeID to login, they cannot set their first/last name and this results in bugs. For now, the easiest way to fix this is to require the review manager to specify the first/last name when inviting users.

## What type of PR is this? (check all applicable)


## Related Tickets & Documents

Fixes [BPHH-530](https://hous-bssb.atlassian.net/browse/BPHH-530)

## Steps to QA

1. Login as Review Manager
2. Go to invite users screen - you should only be able to send the invitation(s) if all fields are filled out for all users

## UI Accessibility Checklist



_Note the following checklist is not an exhaustive list. Check
the [WebAIM checklist](https://webaim.org/standards/wcag/checklist) for
a user-friendly checklist with more details. Aim to meet at least *AA* conformance level where applicable. Check
the [WCAG guidelines](https://webaim.org/standards/wcag/checklist) for the full guidelines._

      with [Wave Browser Extension](https://chromewebstore.google.com/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
      for accessibility errors?

## General Checklist


## [optional] Are there any post-deployment tasks we need to perform?

n/a
